build/deps/textlint.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:lts-alpine
+
+RUN npm install --location=global textlint textlint-rule-one-sentence-per-line
```

Comment:
Let's use packages.json with exact versions for Dependabot too

Reply:
Thank you, fixed